scripts/remove/whisparr.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+app_name="whisparr"
+
+if ! WHISPARR_OWNER="$(swizdb get $app_name/owner)"; then
+    WHISPARR_OWNER=$(_get_master_username)
+fi
+
+user="$WHISPARR_OWNER"
+app_configdir="/home/$user/.config/${app_name^}"
```

Comment:
a swizDB for all of these would be great and allow scripts to be condensed for improved and easy maintainability.